packages/subgraph/tests/admin/admin.test.ts

```diff
@@ -63,7 +63,12 @@ test('Run Admin plugin (handleProposalCreated) mappings with mock event', () =>
   assert.fieldEquals('AdminProposal', entityID, 'id', entityID);
   assert.fieldEquals('AdminProposal', entityID, 'dao', DAO_ADDRESS);
   assert.fieldEquals('AdminProposal', entityID, 'plugin', pluginId);
-  assert.fieldEquals('AdminProposal', entityID, 'proposalId', PROPOSAL_ID);
+  assert.fieldEquals(
+    'AdminProposal',
+    entityID,
+    'pluginProposalId',
+    PROPOSAL_ID
```

Comment:
Should this constant be renamed as well?

Reply:
yes, thanks, amended